projects/angular/src/modal/_modal.clarity.scss

```diff
@@ -126,70 +74,29 @@
     }
 
     .modal-title {
-      @include mixins.css-var(
-        color,
-        clr-modal-title-color,
-        modal-variables.$clr-modal-title-color,
-        variables.$clr-use-custom-properties
-      );
+      color: modal-variables.$clr-modal-title-color;
+      @include mixins.generate-typography-token('SECTION-20-STD');
       margin: 0;
-      @include mixins.css-var(
-        padding,
-        clr-modal-title-padding,
-        0 variables.$clr_baselineRem_0_125,
-        variables.$clr-use-custom-properties
-      );
     }
 
     .modal-title:not([cds-text]) {
-      @include mixins.css-var(
-        font-size,
-        clr-modal-title-font-size,
-        modal-variables.$clr-modal-title-font-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-family,
-        clr-modal-title-font-family,
-        modal-variables.$clr-modal-title-font-family,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-modal-title-font-weight,
-        modal-variables.$clr-modal-title-font-weight,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        line-height,
-        clr-modal-title-line-height,
-        modal-variables.$clr-modal-title-line-height,
-        variables.$clr-use-custom-properties
-      );
+      font-family: modal-variables.$clr-modal-title-font-family;
+      font-size: modal-variables.$clr-modal-title-font-size;
+      font-weight: modal-variables.$clr-modal-title-font-weight;
+      line-height: modal-variables.$clr-modal-title-line-height;
       letter-spacing: modal-variables.$clr-modal-title-letter-spacing;
```

Comment:
Do we have a correct typography from the mixin?

Reply:
We have them and are used in [line 78](https://github.com/vmware-clarity/ng-clarity/blob/shim-removal-wizard_cde-1266/projects/angular/src/modal/_modal.clarity.scss#L78). This rule is to override it when title is `not([cds-text])`.